cmake/Modules/Packages/MLIAP.cmake

```diff
@@ -0,0 +1,3 @@
+execute_process(COMMAND cythonize mliap_model_python_couple.pyx WORKING_DIRECTORY ../src/MLIAP)
```

Comment:
We should look for a way that does not produce files in the source directory `../src/MLIAP`

Reply:
already on it.

Reply:
just pushed what I currently have to see how it fares with integration testing after it is (mostly) working on my local machine. still need to update documentation.